packages/atomic/src/components/commerce/search-box-suggestions/atomic-commerce-search-box-instant-products/atomic-commerce-search-box-instant-products.tsx

```diff
@@ -39,6 +39,7 @@ export type AriaLabelGenerator = (
  *
  * This component is not supported on mobile.
  *
+ * @slot default - The default slot where the instant products are rendered.
```

Comment:
Why this ? This looks out of place.

Reply:
For storybook, we leverage [custom-element-manifest](https://custom-elements-manifest.open-wc.org/analyzer/plugins/intro/) to reflect the capabilities/public API of our components.

While it can infer most stuff from the component code, slots are not one of them, especially the default/unnamed one.

So we declare them explicitly in the JSDOC, so that CEM ingest them, and thus exposes the control in storybook

Reply:
Rule of thumb: If a component can take children, it must have the `@slot default - something` annotation in the JSDOC:

by take children, I mean:
```html
<parent-element>
  <child-element></child-element>
</parent-element>
```

You can detect most components using children elements by the presence of a `<slot/>` element in their render loop